Gemfile

```diff
@@ -21,6 +21,13 @@ gem 'gibberish'
 gem 'hiredis'
 gem 'http_accept_language'
 gem 'identity-hostdata', github: '18F/identity-hostdata', tag: 'v0.4.1'
+require File.join(__dir__, 'app', 'services', 'lambda_jobs', 'git_ref.rb')
```

Comment:
This require is so we can use the same constant `GIT_REF` below as well as in our actual IDP code so that we can use the SHA to resolve the correct lambda to run.... figured this *might* be easier than having the IDP source try to parse the Gemfile.lock or something